index.html

```diff
@@ -0,0 +1,334 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="style.css">
+		<title> My Portfolio </title>
+	</head>
+
+	<body>
+
+		<header id="header">
+			<nav id="navigation">
+				<ul>
+					<li><a href="index.html" id="logo"> My Logo</a></li>
+					<li><a href="" class="menu-link"><img src="./images/iconMenu.svg" alt="menu-icon" class="menu-icon"></a></li>
+				</ul>
+			</nav>
+		</header>
+		<main class="presentation">
+			<div class="greeting">
+				<h1>I’m Mariane Glad to see you!</h1>
+			</div>
+			<div class="self-description">
+				<p>
+					I’m a software developer! I can help you build a product , feature or website Look through some of
+					my work and
+					experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+					me.
+				</p>
+			</div>
+			<a class="btn-mediaConnect" href="#">LET’S CONNECT</a>
+			<ul class="media-list">
+				<li>
+					<a href="" class="media-link"><img src="./images/twitterIcon.svg" alt="twitterIcon" class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/linkedIncon.svg" alt="linkdlnIcon" class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/mailIcon.svg" alt="mailIcon" class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/githubIcon.svg" alt="githubIcon" class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/inconito.svg" alt="inconitoIcon" class="media-icon"></a>
+				</li>
+			</ul>
+		</main>
+
+		<!-- Partie 2 Project Part for review-->
+
+		<main id="section-project">
+			<div class="details-card">
+				<div class="project-image"><img src="./images/project_1.svg" alt="project1img"></div>
+				<div class="card-info">
+					<div class="project-description">
+						<div class="project-details">
+							<h1 class="project-title">Tonic</h1>
```

Comment:
- Kindly, The `h1`  tag indicates a title on a website and it will be one per page , please use `h2`-`h6` based on order, for this one `h2` will be better :+1: